src/js/keyboardInput.js

```diff
@@ -125,6 +126,10 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
             "openTooltipIfNotFocused": {
                 funcName: "gpii.firstDiscovery.keyboardInput.openTooltipIfNotFocused",
                 args: ["{that}", "{that}.container"]
+            },
+            "toggleShiftLatched": {
+                funcName: "gpii.firstDiscovery.keyboardInputTts.toggleShiftLatched",
```

Comment:
Is there a reason that this function uses the namespace of a different component `gpii.firstDiscovery.keyboardInputTts` instead of the component (`gpii.firstDiscovery.keyboardInput`) it's defined for?